src/allotropy/parsers/methodical_mind/methodical_mind_structure.py

```diff
@@ -95,23 +97,41 @@ def create(reader: CsvReader) -> CombinedData:
         )
 
     @staticmethod
-    def _get_parameter(reader: CsvReader, name: str) -> str:
-        return assert_not_none(reader.drop_until_inclusive(name), name).split("\t")[1]
+    def _get_parameter_required(
```

Comment:
Nit: you could write this as
```
def get_parameter(read: CsvReader, name: str, is_required: Optional[bool]=True):
    line = assert_not_none(reader.drop_until_inclusive(name), name).split("\t"))
    content = line[1] if len(line) > 2 else None
     if is_required and not content:
         raise AllotropeConversionError(msg)
     return content
```
And then update above `user = CombinedData._get_parameter(reader, USER, is_required=False)`

Reply:
Good idea, that should work. Thank you!